### ISSUE TITLE
Xenforo not detecting

### ISSUE THREAD

**Issue by maxibanki** [AliasIO/Wappalyzer#1944]
Strangely this is not working although `Forum software by XenForo™` is in the HTML.

Example site:
https://forum.sinusbot.com

Code pointer:
https://github.com/AliasIO/Wappalyzer/blob/67ce37ea5456f35e75273219a034969d4b4be29d/src/apps.json#L9821-L9828

**Comment by gadcam** [comment]
Nice one!
We need to put <code>™</code> and not <code>\&trade;</code> to make it work, could you do a pull request to fix it ?

You can run this code in the console (Ctrl + Shift + K) on the example site to convince you it is the solution
```js
var html = document.documentElement.outerHTML;
var regTrade = new RegExp("Forum software by XenForo&trade;");
var regTM = new RegExp("Forum software by XenForo™")
console.log("Trade: " + regTrade.test(html));
console.log("TM: " + regTM.test(html));
```

**Comment by maxibanki** [comment]
Sure, thanks man!

**Comment by maxibanki** [comment]
Fixed in b11296b60f26fb52c30299a13f56ac1b0fe5f6c6
